cmd/moco-controller/cmd/run.go

```diff
@@ -136,6 +137,13 @@ func subMain(ns, addr string, port int) error {
 		return err
 	}
 
+	if config.pprofAddr != "" {
```

Comment:
I think it would be better to disable pprof even when `0` is specified.
This is because it will be incompatible when using controller-runtime's pprof in the future.

Reply:
That's good. Fixed. 6cf0e1c